lib/NormalModule.js

```diff
@@ -64,6 +64,8 @@ const memoize = require("./util/memoize");
 /** @typedef {import("./ModuleGraph")} ModuleGraph */
 /** @typedef {import("./ModuleGraphConnection").ConnectionState} ConnectionState */
 /** @typedef {import("./NormalModuleFactory")} NormalModuleFactory */
+/** @typedef {import("./NormalModuleFactory").LoaderItem} LoaderItem */
+/** @typedef {import("./NormalModuleFactory").NormalModuleCreateData} NormalModuleCreateData */
```

Comment:
place NormalModuleCreateData inside of `NormalModule.js`

Reply:
done